Eatery Blue/Eatery Blue/UI/EateryScreen/EateryViewController.swift

```diff
@@ -443,11 +443,11 @@ class EateryViewController: UIViewController {
         stackView.addArrangedSubview(categoryView)
     }
 
-    func addReportIssueView() {
+    func addReportIssueView(eateryID: Int) {
```

Comment:
if issues may or may not be about a specific eatery then couldn't eateryId be nil instead?

```suggestion
    func addReportIssueView(eateryId: Int64? = nil) {
```

Reply:
Since CoreData stores ints based on their size I'll also push for using `Int64`s for EateryIDs, and only converting to `Int` when necessary